tests/ObjectConfiguration.tex

```diff
@@ -21,7 +21,7 @@ \subsection{Task Part 1}
 
 	\item \textbf{Analyzing:} Robot views the object configuration in front of it.
 
-	\item \textbf{Description:} Robot describes the configuration to the human collaborator.  The robot can use relational languae or any other means of description.
+	\item \textbf{Description:} Robot describes the configuration to the human collaborator.  The robot can use relational language or any other means of description.
```

Comment:
Double space after the dot.
I haven't clear what is "relational language", meaning that I, as a human, couldn't execute the task without further explanation.